taca-inst.yml

```diff
@@ -4,5 +4,6 @@
   hosts: 127.0.0.1
   connection: local
   roles: 
-   - taca
+   #-ngi_pipeline
+   #- taca
```

Comment:
How are you planning that the installation of `taca` and `ngi_reports` is supposed to happen? Are they going to be in `install.yml`, or should we install them separately via this `taca-inst.yml`? Just wondering because you outcommented these two lines now.